index.html

```diff
@@ -3088,21 +3088,19 @@ <h2 id="OfflineAudioCompletionEvent">
           </dl>
         </section>
         <section>
-          <h3 id="lifetime-AudioNode" class="informative">
+          <h3 id="lifetime-AudioNode">
             Lifetime
           </h3>
           <p>
-            <i>This section is informative.</i>
+            The following behaviors provide a normative description of the
+            conditions under which an AudioNode is alive, meaning that it MUST
+            be retained in the graph by an implementation. Where these
+            conditions do not apply, AudioNodes MAY be released by an
```

Comment:
`<a>AudioNode</a>`s. Maybe it's better to not mention what happens if this node does not have to be retained.